### ISSUE TITLE
MQ statefulset.yaml init container uses mqserver image instead of ibmcom/mq

### ISSUE THREAD

**Issue by meathead23** [IBM/spm-kubernetes#32]
**Issue:**

in `mqservers/templates/statefulsets.yaml` the containers uses the `ibmcom/mq` image from the public IBM repo, whereas the init-containers uses the

container:
```yaml
      containers:
        - name: {{ $.Chart.Name }}-{{ $name }}
          image: ibmcom/mq:{{ $.Values.global.mq.version }}
```

init-container:
```yaml
      initContainers:
      - name: {{ $.Chart.Name }}-{{ $name }}-init
        image: {{ include "mqserver.imageFullName" $.Values.global.images }}
```

Shouldn't these be the same, the instructions in https://ibm.github.io/spm-kubernetes/03-build-images/build_images don't mention building or pushing the mqserver image. Because if this i'd get errors on pulling the init containers.

**Solution:**

Change the image to:

```yaml
      initContainers:
      - name: {{ $.Chart.Name }}-{{ $name }}-init
        image: ibmcom/mq:{{ $.Values.global.mq.version }}
```

Once this is done the init container pulls OK.

**Comment by mhcastro** [comment]
@meathead23 thanks. Yes, we've noticed this issue and it will be fixed in the next release.

**Comment by dobharweim** [comment]
@meathead23 we have removed the `initContainers` from `mqservers/templates/statefulsets.yaml` and are handling the functionality they used to provide within the `containers` spec itself, so this issue is no longer relevant. 

Thanks though it was a great catch.
